src/models/lineChart.js

```diff
@@ -383,7 +383,8 @@ nv.models.lineChart = function() {
                 var singlePoint, pointIndex, pointXLocation, allData = [];
                 data
                     .filter(function(series, i) {
-                        series.seriesIndex = i;
+                        // Assign the index here instead of in the filter
+                        // as the DOM removes disabled series
```

Comment:
Whoops, sorry I messed up here when removing the build files it should be doing `series.seriesIndex = i;` inside the forEach. Will fix.